### ISSUE TITLE
Android project's configuration `manifestPath` is not used by @react-native-community/cli-platform-android

### ISSUE THREAD

**Issue by jussikinnula** [react-native-community/cli#1159]
## Environment

```
info Fetching system and libraries information...
System:
    OS: macOS 10.15.4
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Memory: 17.21 GB / 64.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.22.4 - ~/.nvm/versions/node/v12.16.1/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
    Watchman: Not Found
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.3
      System Images: android-29 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6392135
    Xcode: 11.4.1/11E503a - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_252 - /Users/jami/.sdkman/candidates/java/current/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: ^4.8.0 => 4.8.0
    react: 16.11.0 => 16.11.0
    react-native: 0.62.2 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```
## Description

Android project's configuration `manifestPath` is not used by `@react-native-community/cli-platform-android` phase `buildAndRun`.

## Reproducible Demo

https://github.com/jussikinnula/react-native-webview/tree/feat-react-native-0-62-support

Just run `yarn start:android` and see. Note that React Native is upgraded to 0.62, which changed the configuration it interprets.

The actual problem is that react-native-webview has example in a separate folder called `example/` in root, instead of default directly in root. Current version of @react-native-community/cli-platform-android uses hardcoded path for manifest, which works only in the simple case.

**Comment by jussikinnula** [comment]
See proposed fix from pull request: https://github.com/react-native-community/cli/pull/1160

**Comment by jamonholmgren** [comment]
I believe this has been fixed by #1160 -- should we close this?

**Comment by thymikee** [comment]
Yup! Thanks @jamonholmgren
